tests/testthat/_snaps/expect-constant.md

```diff
@@ -1,34 +1,34 @@
 # logical tests act as expected
 
-    FALSE (`actual`) is not equal to TRUE (`expected`).
+    FALSE is not equal to TRUE.
```

Comment:
Should we quote the expressions?


```suggestion
    `FALSE` is not equal to `TRUE`.
```

I guess this becomes too heavy in the parenthetised/string-quoted cases

```
    x` (`1`) is not identical to `"a"`.
```